.pre-commit-config.yaml

```diff
@@ -100,6 +100,57 @@ repos:
         pass_filenames: false
         language: system
 
+      # Javascript linting, formatting.
+      # We're using the local node_modules to simplify things, otherwise would
+      # need an easy way to keep versions in sync. Furthermore some repositories
+      # are either not on pre-commit mirrors or are missing some tagged versions.
+      # pre-commit-managed nodeenv environments also do not work out of the box because
+      # additional_dependencies does not install a flattened dependency tree which is needed by,
+      # for example, our eslint setup.
+
+      - id: biome
+        name: biome (javascript, typescript, json)
+        language: system
+        types: [text]
+        files: \.(jsx?|tsx?|css)$
```

Comment:
Maybe `types_or` would be better here? Not a strong opinion

Reply:
oh, will just remove this entirely, `files` there is fine + we have that for most of the other hooks